src/app.module.ts

```diff
@@ -1,9 +1,37 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import {ConfigModule, ConfigService} from '@nestjs/config'
+import {TypeOrmModule} from '@nestjs/typeorm'
+import { UsersModule } from './users/users.module';
+import join from 'path'
+import configuration from './config/configuration'
+import {SnakeNamingStrategy} from 'typeorm-naming-strategies'
 
 @Module({
-  imports: [],
+  imports: [
+    ConfigModule.forRoot({
+      envFilePath: '.env.local',
+      isGlobal: true,
+      load: [configuration],
+    }),
+    TypeOrmModule.forRootAsync({
+      imports: [ConfigModule],
+      inject: [ConfigService],
+      useFactory: (configService: ConfigService) => ({
+        type: 'mysql',
```

Comment:
@colin950 노션 저희 토이프로젝트 팀 문서에 윤호님이 Postgresql RDS 하나 개설하셔서 공유하신 게 있습니다!
postgresql 용으로 수정하면 좋을 것 같습니당.

```suggestion
        type: 'postgres',
```

Reply:
넵